.github/workflows/upload-win-installer.yml

```diff
@@ -98,20 +101,19 @@ jobs:
           echo "APP_ID=${{secrets.AZ_APP_ID}}" | Out-File -FilePath $env:GITHUB_ENV -Append
           echo "TENANT_ID=${{secrets.AZ_TENANT_ID}}" | Out-File -FilePath $env:GITHUB_ENV -Append
           echo "CLIENT_SECRET=${{secrets.AZ_CLIENT_SECRET}}" | Out-File -FilePath $env:GITHUB_ENV -Append
+    - name: Pandoc Setup
+      uses: r-lib/actions/setup-pandoc@v2
+      with:
+        pandoc-version: '3.1.11'
     - name: Build
       id: build
       if: steps.check.outputs.already-exists != 'true' || steps.actual_dryrun.outputs.dryrun == 'true'
       run: |
-        Push-Location contrib\win-installer
-        .\build.ps1 ${{steps.getversion.outputs.version}} prod
-        $code = $LASTEXITCODE
-        if ($code -eq 2) {
-          Write-Output "artifact-missing=true" | Out-File -FilePath $env:GITHUB_OUTPUT -Append
-          Pop-Location
-          Exit 0
-        }
-        Pop-Location
-        Exit $code
+        .\winmake.ps1 podman
+        .\winmake.ps1 podman-remote
```

Comment:
`podman` is a `podman-remote` alias. It's built twice the same binary here.
```suggestion
```

Reply:
Done, thanks!